File: MotivApp/app/src/main/java/com/motiv/piotr/MainActivityContract.java
package com.motiv.piotr;


public interface MainActivityContract {

    public interface View {

        public java.lang.CharSequence edittext10getText();

        public void showToast(java.lang.String arg0);

        public void navigationControllerstartDashboardActivity(com.motiv.piotr.User arg0);
    }

    public interface Presenter {

        public void eloonClick(android.view.View argument0);
    }
}

File: MotivApp/app/src/main/java/com/motiv/piotr/MainActivityContract.kt
 
package com.motiv.piotr

public interface MainActivityContract {

    public interface View {

        public fun edittext10getText(): CharSequence
        public fun showToast(arg0: String): Unit
        public fun navigationControllerstartDashboardActivity(arg0: com.motiv.piotr.User): Unit
    }

    public interface Presenter {

        public fun eloonClick(argument0: android.view.View): Unit
    }
}
